Rakefile

```
@@ -19,4 +20,11 @@ task :clean do
   Jekyll::Commands::Clean.process({})
 end
 
+# Test generated output has valid HTML and links.
+task :test do
+  Rake::Task["build"].execute

```

I think the correct way to do this is:

```suggestion
task :test => :build do
```

---

Yes, that also works.